zerver/actions/create_user.py

```diff
@@ -535,32 +545,32 @@ def do_activate_mirror_dummy_user(
     notify_created_user(user_profile)
 
 
+@transaction.atomic(savepoint=False)
```

Comment:
Cannot use `durable=True` here, since `do_reactivate_user` is called inside from `save` function in `zerver/lib/scim.py` and looking at `django-scim` code, `save` function is already called inside a transaction and transaction with `durable=True` should be outermost transaction.